src/types/interest_rate.ts

```diff
@@ -1,9 +1,24 @@
 import { BigNumber } from "../../utils/bignumber";
 
 export class InterestRate {
+    public static fromRaw(value: BigNumber): InterestRate {
```

Comment:
I think something like `fromBigNumber` would be more explicit.

Reply:
I think we should discuss, but we've been using raw to represent the BigNumber amount stored on the blockchain. See TokenAmount. Think we should keep this consistent.